hubmap/query_app/validation.py

```diff
@@ -400,3 +403,38 @@ def validate_bounds_args(query_params: Dict):
             and Gene.objects.filter(gene_symbol=query_params["var_id"]).first() is None
         ):
             raise ValueError(f"{query_params['var_id']} is not in gene index")
+
+
+def validate_input_terms(input_type: str, input_set: List[str]):
+
+    input_set = [
+        split_at_comparator(item)[0].strip()
+        if len(split_at_comparator(item)) > 0
+        else item.strip()
+        for item in input_set
+    ]
+
+    identifiers_not_found = []
+    if input_type == "gene":
```

Comment:
It seems like this could be shortened with something like
```python
input_type_model_mapping = {
    "gene": (Gene, "gene_symbol"),
    "protein": (Protein, "protein_id"),
    "organ": (Organ, "grouping_name"),
    ...
}

model, kwarg_piece = input_type_model_mapping[input_type]
identifiers_not_found = [
    item for item in input_set if not model.objects.filter(**{f"{kwarg_piece}__iexact": item})
]
```
I'm not sure how necessary I think that is -- though I did have to look through the different `if input_type == ...` checks in detail to check whether the different blocks were actually the same, so it probably is worth consolidating this make it clear that these are all conceptually the same check.